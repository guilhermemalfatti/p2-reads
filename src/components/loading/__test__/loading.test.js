import React from 'react';
import Loading from '../index';
import renderer from 'react-test-renderer';

test('Loading render snapshot', () => {
    const component = renderer.create(
        <Loading ></Loading>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});