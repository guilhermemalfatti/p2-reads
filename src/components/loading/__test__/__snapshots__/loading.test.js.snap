// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render snapshot 1`] = `
<div
  className="loader"
/>
`;
